src/views/Terms/__tests__/TermList.test.js

```diff
@@ -58,16 +59,19 @@ describe('TermList component rendered with English', () => {
 				dictionaryTitle,
 				language,
 				searchBoxTitle,
+				canonicalHost: "https://cancer.gov"
 			},
 		]);
 
 		await act(async () => {
 			wrapper = render(
```

Comment:
minor - looks like you need to format , missing tabs after on MemoryRouter everywhere you added Mock Analytics provider